### ISSUE TITLE
Multiplying 32F image with a scalar crashes

### ISSUE THREAD

**Issue by viktor-ferenczi** [hybridgroup/gocv#277]
Code:

```
img := gocv.IMRead(path, gocv.IMReadAnyColor)
scale := 0.5
scalar := gocv.Scalar{scale, scale, scale, scale}
scalarMat := gocv.NewMatFromScalar(scalar, img.Type())
gocv.Multiply(img, scalarMat, &img)

```

Results in crash:

```
Exception 0x20474343 0x171a330 0x64945138 0x7ffd159ca388
...
goroutine 5 [syscall]:
gocv.io/x/gocv._Cfunc_Mat_Multiply(0x161afd0, 0x171c680, 0x161afd0)
	_cgo_gotypes.go:2907 +0x4c
```

No meaningful error message. (Like in case of mismatching dimensions or channels...)

How does the scaling with a scalar supposed to work in Go?

- Operating System and version: Windows 10 Pro 64bit
- OpenCV version used: 3.4.2
- How did you install OpenCV: Followed Windows installation instructions
- GoCV version used: 0.15.0
- Go version: 1.10.3
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`? No - But, is this still relevant?

Suboptimal workaround:

```
img := gocv.IMRead(path, gocv.IMReadAnyColor)
scale := 0.5
gocv.AddWeighted(img, scale, img, 0.0, 1.0, &img)
```

**Comment by deadprogram** [comment]
That function requires Mat values as params.

https://github.com/hybridgroup/gocv/blob/3f4d73bc6a6e20e4880c0ea208f3e80880e1c725/core.go#L1257

also 

https://docs.opencv.org/master/d2/de8/group__core__array.html#ga979d898a58d7f61c53003e162e7ad89f

The GoDoc comment should be updated to match the OpenCV docs comments.

**Comment by deadprogram** [comment]
Hi @viktor-ferenczi 

I've just updated the GoDoc and test for `Multiply()` in the `dev` branch.

Please see https://github.com/hybridgroup/gocv/commit/508479aac81f33aa70e0a15947b8c6aadeb97684

Hopefully this clarifies things a bit.

I will close this issue once this code goes out as part of the next release, unless you have any objections.

Thanks for pointing this out.

**Comment by deadprogram** [comment]
This has been released as part of v0.16.0 thank you.
